platform/app/src/routes/WorkList/WorkList.tsx

```diff
@@ -341,6 +345,10 @@ function WorkList({
 
             const modalitiesToCheck = modalities.replaceAll('/', '\\');
 
+            if (mode.noWorklistButton === true) {
```

Comment:
- noWorklistButton is to disable rendering a mode button here if I want to have a hidden mode or something like that:

![Screenshot 2023-06-13 at 13 49 03](https://github.com/OHIF/Viewers/assets/13886933/0930c1f1-de2b-4959-b4b9-e4a9595a885c)
- isReturnEnabled was already there I just exposed it so people can disable the return to worklist button per mode instead of a global config option (current implementation)

Reply:
Please add `noWorklistButon` to the table of mode properties found in this [section](https://docs.ohif.org/platform/modes/#anatomy) of our docs. However, I am not sure if this is any different from simply not including the mode in the config file from the get-go. What is the use case for this? Would it be to include the mode (for URL viewer launch), but exclude it in the worklist?

Reply:
Also consider where to document `isReturnEnabled`. Maybe in the docs for [Layout template](https://docs.ohif.org/platform/extensions/modules/layout-template#overview)? We do mention the default extension ViewerLayout there, but I am not sure that is the best place. If we are adding it then we should have people know about it. Just a suggestion. :)

Reply:
@jbocce Yes, the idea is to have a mode that does not render a button from a worklist e.g. a URL only mode. This way I don't have to build an URL for gcp from a worklist.

Reply:
Updated, thanks!

Reply:
Thanks for updating the docs. However, I noticed that upload is still broken. But the good news is that I think it can be simply fixed by adding a null check on line 453 as such...

```
dicomUploadComponent && dataSource.getConfig()?.dicomUploadEnabled
```

Reply:
Sorry, what do you mean by broken? i couldn't see anything in the logs

Reply:
As I briefly mentioned in my first review [here](https://github.com/OHIF/Viewers/pull/3427#discussion_r1231449990), if you try to run the `config/local_orthanc.js` or `config/local_dcm4chee.js` you will notice that OHIF does not even run at all.

Reply:
Both of those data source configurations have dicom upload enabled.